config/testgrids/istio/istio.yaml

```diff
@@ -0,0 +1,1222 @@
+default_dashboard_tab:
+  attach_bug_template: {}
+  code_search_path: github.com/istio/istio/search
```

Comment:
Default settings won't be respected (see #13636 )

The newest version of Configurator should regenerate these files without the defaults. Alternately, please delete lines 1-29